android/app/src/main/res/values-land/dimens.xml

```diff
@@ -12,4 +12,9 @@
   <dimen name="altitude_chart_image_width">334dp</dimen>
 
   <dimen name="start_button_width">128dp</dimen>
+
+  <!--About Fragment -->
+  <dimen name="max_healine_width">260dp</dimen>
+  <dimen name="max_bullet_width">280dp</dimen>
+  <dimen name="max_button_width">350dp</dimen>
```

Comment:
```suggestion
  <dimen name="about_max_button_width">350dp</dimen>
```

Reply:
@Jean-BaptisteC please don't use generic identifiers, always specialize where are they used (here, add about_ prefix for example).

Reply:
Yes, I don't have update this now, because I have planned to work on buttons and maybe remove hardcoded values